packages/notifi-dapp-example/src/components/AlertSubscription.tsx

```diff
@@ -1,21 +1,77 @@
 'use client';
 
+import { FusionEventTopic } from '@notifi-network/notifi-frontend-client';
 import { useNotifiTenantConfigContext } from '@notifi-network/notifi-react';
+import React from 'react';
 
 import { AlertSubscriptionRow } from './AlertSubscriptionRow';
 
-export type AlertSubscriptionRowProps = {
+export type AlertSubscriptionProps = {
   title?: string;
   inFTU?: boolean;
 };
 
-export const AlertSubscription: React.FC<AlertSubscriptionRowProps> = ({
+export type AlertSubscriptionTopicRowCategory = 'standalone' | 'group';
+
+export type AlertSubscriptionTopicRowMetadata<
+  T extends AlertSubscriptionTopicRowCategory,
+> = T extends 'standalone'
+  ? AlertSubscriptionTopicStandaloneRowMetadata
+  : AlertSubscriptionTopicGroupRowMetadata;
+
+type TopicRowMetadataBase = {
+  index: number;
+};
+
+export type AlertSubscriptionTopicStandaloneRowMetadata =
+  TopicRowMetadataBase & {
```

Comment:
Just a nit, we might want to make names as consistent as possible since this is an "example" repo. So we do not want to confuse those devs using the repo. 
There are few naming here:
1. AlertSubscriptionTopicRow --> component name
2. AlertSubscriptionRow --> file name
3. TopicRow --> Sometimes show across the code base.
How do you think to make those name the same, could be simplified only `TopicRow`??